types/react-native/index.d.ts

```diff
@@ -4974,10 +4975,19 @@ export interface TouchableWithoutFeedbackPropsIOS {
     tvParallaxProperties?: TVParallaxProperties;
 }
 
+export interface TouchableWithoutFeedbackPropsAndroid {
+    /**
+     * If true, doesn't play a system sound on touch.
+     *
+     * @platform android
+     */
+    touchSoundDisabled?: boolean;
```

Comment:
Technically the upstream Flow typing is:

```suggestion
    touchSoundDisabled?: boolean | null;
```

Reply:
A Flow type prefixed with a question mark is a “maybe” type. Both `undefined` and `null` are assignable to Flow’s “maybe” types: https://flow.org/en/docs/types/maybe/

Reply:
@allow i see but disabled prop has the same prefixed question mark. So should we change it as well then ?

Reply:
> @Allow i see but disabled prop has the same prefixed question mark. So should we change it as well then ?

Yup, please do 👍 

To be honest, there are probably more cases that should be covered, but for now I mostly want to make sure we don’t introduce any new cases.

Reply:
@alloy done

Reply:
Great, thanks!